[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:36,286 INFO [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:219] - [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。] Unexpected exception during request: [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。]
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:36,286 INFO [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:219] - [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。] Unexpected exception during request: [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。]
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:36,762 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:36,762 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:38,907 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:38,910 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:43,149 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:43,149 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:45,197 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:45,212 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:49,744 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:50,150 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:51,964 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:52,370 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:55,341 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:57,644 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:57,970 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:47:59,870 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:02,141 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:04,443 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:05,348 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:07,565 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:11,753 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:12,929 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:13,820 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:15,041 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:22,050 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:23,252 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:24,094 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:25,298 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:40,548 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:41,742 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:42,931 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:48:43,789 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:13,003 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:13,850 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:15,057 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:15,879 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:45,143 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:45,941 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:47,188 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:49:48,000 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:17,246 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:18,046 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:19,290 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:20,075 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:49,351 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:50,144 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:51,372 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:50:52,193 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:51:21,441 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:51:22,249 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:51:23,471 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2022-12-09 19:51:24,274 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
